### DISCUSSION TITLE
Auto Startup

### DISCUSSION THREAD

**Discussion by swartjie** [NativePHP/laravel#228] [Ideas]
Hey Guys,

Can you start up an app built with NativePHP automatically on windows boot?
Also, How does one go about the installer? Is there a way to create an installer that can install dependencies etc aswell or what do you recommend?

Would love some advise?

**Comment by simonhamp** [comment]
I've moved this to the Ideas category as it's a good one. We don't currently support this, but [Electron has this baked in](https://www.electronjs.org/docs/latest/api/app#appsetloginitemsettingssettings-macos-windows), so we could do!

**Reply by WINBIGFOX** [reply]
Are there any plans for when this feature will be available?

**Reply by simonhamp** [reply]
Probably post-v1's general release
